roles/puppet_repositories/README.md

```diff
@@ -0,0 +1,36 @@
+theforeman.operations.puppet_repositories
+=========================================
+
+Sets up Puppet repositories
+
+Role Variables
+--------------
+
+Optional:
+
+- `puppet_repositories_version`: Version of Foreman to setup repositories for (default: 2.3)
```

Comment:
Uhm, no, I don't think this will setup Foreman 2.3 repos ;)

Reply:
It might if you wish hard enough.